METS-Programme/openmrs-module-ugandaemr
# Assignment of clients to Refill-Groups or Pharmacies depending on their DSD Model

How to test 
-Administrator is able to register register cohorts of a particular Cohort type via Home>System administration >DSD SubGroup Configuration
-Enable CDDP Categorization in ugandaemr global properties
-On the encounter page under DSD tab, once a patient is able to go on CDDP, you can further categorise him under CRPDDP or CLDDP. On picking any of the groups, the DSD Group-Refill Point drop down fills with the cohorts registered under that sub category 
-The unique identifier of that cohort is displayed in the DSD Group Code field 
-On saving the encounter a client is added to the assigned  cohort and identifier code also saved as an obs 
Note : This PR is built on top of https://github.com/METS-Programme/openmrs-module-ugandaemr/pull/100 which is implemented not  based Cohort assignment but rather on Locations
